test/google_crawler_web/features/upload/upload_file_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule GoogleCrawlerWeb.UploadFileTest do
+  use GoogleCrawlerWeb.FeatureCase
+
+  @selectors %{
+    file_field: "upload[file]"
+  }
+
+  feature "upload a file", %{session: session} do
```

Comment:
```suggestion
  feature "uploads a file", %{session: session} do
```

Reply:
Fixed in https://github.com/ankitkalia1195/elixir-google-crawler/pull/34/commits/f300517565b4673169c7ca8270285d66e2c1cbfe